src/utils/dispute-utils.js

```diff
@@ -393,8 +394,13 @@ function getRoundPhasesAndTime(courtConfig, round, currentPhase) {
 
   const roundPhasesAndTime = [
     {
-      // Jurors can be drafted at any time
+      // Jurors can be drafted at any time, so we'll only set the
+      // `endTime` when the drafting phase has already passed
       phase: DisputesTypes.Phase.JuryDrafting,
+      endTime:
+        DisputesTypes.Phase.JuryDrafting !== currentPhase.phase
```

Comment:
Just wondering, how do we know the later steps' end times if we haven't got past the draft phase?

Should we do an early return here in that case?